apps/dashboard/app/models/announcement.rb

```diff
@@ -31,10 +32,36 @@ def msg
     msg.blank? ? nil : msg
   end
 
+  def button_text
+    opts.fetch(:button_text, I18n.t('dashboard.announcements_button_text')).to_s
```

Comment:
I feel like this should be 2 things, one for `dismissable`, and another for `required`.

Required should be something like `Accept` and `dismissable` maybe, `Don't show again`.

Reply:
🤦‍♂️ I see now that this is overridable. Even so, maybe we can still supply two defaults depending, just for convenience so they don't have to always override the button text.

Reply:
I added 2 default button texts, one for required and one for dismissible